tests/unit/customizations/emr/test_add_steps.py

```diff
@@ -212,7 +212,7 @@ def test_custom_jar_step_with_all_fields(self):
             'Jar=s3://mybucket/mytest.jar,'
             'Args=arg1,arg2,MainClass=mymainclass,'
             'ActionOnFailure=TERMINATE_CLUSTER,'
-            'Properties=k1=v1\,k2=v2\,k3')
+            r'Properties=k1=v1\,k2=v2\,k3')
```

Comment:
This is a bit of a nit, but when we have a multi-line string, we may want to lean toward prefixing everything to make sure it's consistent/understood this is one raw string.